features/steps/__init__.py

```
@@ -0,0 +1,10 @@
+import os
+import pkgutil
+
+__all__ = []
+PATH = [os.path.dirname(__file__)]
+
+for loader, module_name, is_pkg in pkgutil.walk_packages(PATH):
```

Ruff (0.8.0)

Loop control variable `is_pkg` may not be used within loop body

(B007)